src/ert/run_models/run_arguments.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class SingleTestRunArguments:
+    simulation_mode: str = field(init=False, default="Single test run")
```

Comment:
Unused afaik, but:

```suggestion
    simulation_mode: Literal["Single test run"] = "Single test run"
```

This means that the value can only ever be the literal value. It's also useful in Pydantic (for future reference):

```py
class Foo(BaseModel):
  kind: Literal["Foo"] = "Foo"
  intvalue: int

class Bar(BaseModel):
  kind: Literal["Bar"] = "Bar"
  strvalue: str

class MyData(BaseModel):
  foo_or_bar: Union[Foo, Bar]

MyData.parse_raw('{"foo_or_bar": {"kind": "Foo", "intvalue": 42}}')  #=> OK
MyData.parse_raw('{"foo_or_bar": {"kind": "Bar", "strvalue": "some_str"}}')  #=> OK
MyData.parse_raw('{"foo_or_bar": {"kind": "Foo", "strvalue": "some_str"}}')  #=> Error! `Foo` does not have `strvalue`
```

Eivind says that we can use `from pydantic.dataclasses import dataclass` instead of `from pydantic import BaseModel` to have the same API as the built-in `dataclass`, but using Pydantic features like validators. I haven't used this so I leave it as an excercise to the reader.

Reply:
Did not know this was unused, and thanks for the trick 👍